### ISSUE TITLE
omitting entries in the service.json file creates invalid service file

### ISSUE THREAD

**Issue by ben-page** [simone-sanfratello/node-service-systemd#5]
I don't have environment variable that need to be set, so I removed the item from my service.json file. This let to the service file being created improperly.

Notice, the erroneous `{environment}`

```
[Unit]
Description=verdaccio service
Requires=network.target
After=network.target

[Service]
Type=forking
WorkingDirectory=/var/www/verdaccio
{environment}ExecStart=/usr/local/bin/systemd-verdaccio-start
ExecStop=/usr/local/bin/systemd-verdaccio-stop
PIDFile=/var/www/verdaccio/verdaccio.pid
User=www-data
Group=www-data

[Install]
WantedBy=multi-user.target
```

**Comment by simone-sanfratello** [comment]
set "environment" as emèpty string; I will fix

**Comment by simone-sanfratello** [comment]
fixed on 2.0.2
